frida-gum/src/range_details.rs

```diff
@@ -0,0 +1,60 @@
+use frida_gum_sys as gum_sys;
+use std::ffi::CStr;
+use std::os::raw::c_void;
+
+use crate::{NativePointer, MemoryRange};
+
+pub struct FileMapping {
+    file_mapping: gum_sys::GumFileMapping,
+}
+
+impl FileMapping {
+    pub fn new_from_raw(file: gum_sys::GumFileMapping) -> Self {
```

Comment:
This should be `from_raw` to be consistent. It should also not be a public interface. You probably want `pub(crate)`

Reply:
ok.